components/DragContainer.js

```diff
@@ -31,17 +31,27 @@ export default function DragContainer({
   );
 }
 
-const Container = styled.section`
+const Container = styled.ul`
   position: relative;
-  display: flex;
   margin-bottom: 10px;
+  --padding: 3px 0;
   width: 100%;
-  height: 12rem;
-
+  --height: 11rem;
+  --background: #ccc;
+  display: flex;
+  list-style: none;
+  overflow-x: auto;
+  overflow-y: hidden;
   &:empty:not(:focus):before {
```

Comment:
don't know what all of that is, but it looks impressive&confusing at the same time

Reply:
i just copy that from google. It s the drag drop thing, simulating placeholder effect in input or textarea